tests/test_cleaneval.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/env python2.5
+# encoding: utf-8
+"""
+cleaneval.py v1.0
```

Comment:
this string needs updating from the original; you should provide a link to the original